.gitignore

```diff
@@ -161,4 +161,4 @@ node_modules
 .node_repl_history
 
 # Build folder
-dist
+# dist
```

Comment:
why do u remove dist folder from ignore?

Reply:
sorry, this comes with a commit I pushed on my master because I did not push on npm, I'm referencing this package directly from github repo.

I opened [a new PR](https://github.com/chantouchsek/v-sanitize/pull/108) without this change from a branch I created on purpose